README.md

```diff
@@ -35,14 +35,16 @@ https://demo.sketchbookjs.com
 3.  Run on `http://localhost:3000/`:
 
     ```
-    yarn sketchbook start
+    yarn sketchbook start <path/to/design/file.json>
```

Comment:
I think it'd be better to just use the generated file here:

```suggestion
    yarn sketchbook start sketchbook/example.json
```

That way someone can just copy and paste the commands from each step and get something up and running without having to think about it too much.

Reply:
Fixed @haydn